### ISSUE TITLE
USER STORY: Model training, optimization and validation.

### ISSUE THREAD

**Issue by md-ash-dot** [md-ash-dot/mildew-detector#4]
As a **data practitioner** I can **train, optimize and validate a model** so that **I can predict if a given cherry leaf is a healthy or powdery mildew contained cherry leaf.**

- Check the number of images in train, test and validation data.

- Augment training, validation, test image dataset.

- Create a model.

- fit the model for training.

- Check model performance by plotting the model learning curve.

- Evaluate the model on the test set.

- Adjust hyperparameters and optimize the model if needed.

- Predict on new data.
